gr-blocks/lib/add_const_ff_impl.h

```diff
@@ -32,14 +32,20 @@ class BLOCKS_API add_const_ff_impl : public add_const_ff
 {
 private:
     float d_k;
+    volk::vector<float> d_k_copy;
 
 public:
     add_const_ff_impl(float k);
 
     void setup_rpc();
 
     float k() const { return d_k; }
-    void set_k(float k) { d_k = k; }
+
+    void set_k(float k);
+
+    volk::vector<float> k_copy() const { return d_k_copy; }
```

Comment:
Do we really need to return the vector? I can't think of a case where this is useful, but I'm open to suggestions. If there are cases, then I'd suggest using `const volk::vector<float>&` as the return type.